include/swift/Frontend/ReferenceDependencyKeys.h

```diff
@@ -0,0 +1,39 @@
+//===--- ReferenceDependencyKeys.h - Keys for swiftdeps files ---*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_FRONTENDTOOL_REFERENCEDEPENDENCYKEYS_H
+#define SWIFT_FRONTENDTOOL_REFERENCEDEPENDENCYKEYS_H
+
+#include "swift/Basic/LLVM.h"
+#include "llvm/ADT/ArrayRef.h"
+#include <string>
+#include <vector>
+
+namespace swift {
+  namespace reference_dependency_keys {
+    constexpr StringLiteral topLevelProviders("provides-top-level");
+    constexpr StringLiteral nominalProviders("provides-nominal");
+    constexpr StringLiteral memberProviders("provides-member");
+    constexpr StringLiteral dynamicLookupProviders("provides-dynamic-lookup");
+    
+    constexpr StringLiteral topLevelDependencies("depends-top-level");
+    constexpr StringLiteral memberDependencies("depends-member");
+    constexpr StringLiteral nominalDependencies("depends-nominal");
+    constexpr StringLiteral dynamicLookupDependencies("depends-dynamic-lookup");
+    constexpr StringLiteral externalDependencies("depends-external");
+    
+    constexpr StringLiteral interfaceHash("interface-hash");
```

Comment:
These still need to be static.

Reply:
Huh. I fixed this but someone Tower's tracking stopped and it didn't get pushed. Sorry.

Reply:
Tx fr catching it.